### ISSUE TITLE
Uncaught TypeError: Cannot read property 'insertBefore' of null

### ISSUE THREAD

**Issue by darrennickerson** [christophery/pushy#88]
I'm getting an error with pushy, pushy.js?ver=4.5.1:115 Uncaught TypeError: Cannot read property 'insertBefore' of null.

**Comment by hoverlover** [comment]
I fixed it locally by doing this:

```diff
@@ -112,18 +112,22 @@
                };

                // Add it to the body to get the computed style
-               document.body.insertBefore(el, null);
-
-               for(var t in transforms){
-                   if( el.style[t] !== undefined ){
-                       el.style[t] = 'translate3d(1px,1px,1px)';
-                       supported = window.getComputedStyle(el).getPropertyValue(transforms[t]);
-                   }
-               }
+               if (document.body !== null) {
+                       document.body.insertBefore(el, null);
+
+                       for(var t in transforms){
+                           if( el.style[t] !== undefined ){
+                               el.style[t] = 'translate3d(1px,1px,1px)';
+                               supported = window.getComputedStyle(el).getPropertyValue(transforms[t]);
+                           }
+                       }

-               document.body.removeChild(el);
+                       document.body.removeChild(el);

-               return (supported !== undefined && supported.length > 0 && supported !== "none");
+                       return (supported !== undefined && supported.length > 0 && supported !== "none");
+               } else {
+                       return false;
+               }
        })();

        if(cssTransforms3d){
```

I'll be happy to submit a PR for this, but I'd like the author to comment so I can know if it's a proper fix in his eyes.

**Comment by christophery** [comment]
@hoverlover Looks good to me! Feel free to submit a PR.

**Comment by BrianHotopp** [comment]
@hoverlover I had my js attatched in the head instead of after the body. Fixed my issue.

**Comment by Overtrainer** [comment]
Just add tag <script type="text/javascript" src="js/pushy.min.js"></script> in fotter, or wrap pushy.min.js in strings $(function() { // here is all pushy.min.js code  })

**Comment by rotanadan** [comment]
This still seems to be an issue?

**Comment by LuxDigital** [comment]
Why has this not been merged?

**Comment by christophery** [comment]
Fixed w/ #107
